docs/tutorial_bracket_strategy.md

```diff
@@ -135,14 +139,14 @@ export PK=<the private key of the proposer account>
 ```ssh
 cd dex-liquidity-provision
 yarn install
-npx truffle compile
+yarn compile
 yarn run networks-inject
-npx truffle exec scripts/complete_liquidity_provision.js --targetToken=0 --stableToken=7 --lowestLimit=150 --highestLimit=260 --currentPrice=200 --masterSafe=$MASTER_SAFE --investmentTargetToken=5 --investmentStableToken=1000 --fleetSize=20 --network=$NETWORK_NAME
+npx truffle exec scripts/complete_liquidity_provision.js --baseTokenId=0 --quoteTokenId=7 --lowestLimit=150 --highestLimit=260 --currentPrice=200 --masterSafe=$MASTER_SAFE --depositBaseToken=5 --depositQuoteToken=1000 --fleetSize=20 --network=$NETWORK_NAME
```

Comment:
This looks like a bracket strategy with OWL. Should we maybe use tokens 1 and 7 (WETH and DAI) here?